docs/getting-started.md

```diff
@@ -78,9 +78,13 @@ You can now run a mining node:
 radicle-registry-node --miner <address>
 ```
 
-`TODO: You can see mining in the logs`
+If your node successfully mined a block you will see a log line like this
+```
+15:50:13.041 INFO radicle_registry_node::service  Imported own block #1322 (0xefd1…4445)
```

Comment:
I'd make it more clear that it's about the "Imported _own_ block", but leave that with you.

Reply:
Do you have a suggestion on how to make this clearer? How about
> If your node successfully mined a block and imported it will log “Imported own block”:
> 15:50:13.041 INFO radicle_registry_node::service  Imported own block #1322 (0xefd1…4445)

Reply:
yeah, that's good :+1: